Source/DafnyDriver/DafnyDriver.cs

```diff
@@ -505,6 +507,40 @@ public static CommandLineArgumentsResult ProcessCommandLineArguments(string[] ar
 
     class DafnyConsolePrinter : ConsolePrinter
     {
+      private readonly Dictionary<string, List<string>> fsCache = new Dictionary<string, List<string>>();
+
+      private string GetFileLine(string filename, int lineNumber) {
+        List<string> lines;
+        if (!fsCache.ContainsKey(filename)) {
+          try {
+            // Note: This is not guaranteed to be the same file that Dafny parsed. To ensure that, Dafny should keep
+            // an in-memory version of each file it parses.
+            lines = File.ReadLines(filename).ToList();
+          } catch (Exception _) {
```

Comment:
Could you try to avoid the reported warning?

Reply:
Done